python/lsst/solsys/pipe/makeTracklets.py

```diff
@@ -1,129 +1,173 @@
+        name="dia_source_dayobs_14"
     )
-    sspVisitInputs = connectionTypes.PrerequisiteInput(
+    sspVisitInputs = connectionTypes.Input(
         doc="visit stats plus observer coordinates",
-        dimensions=["instrument"],
+        dimensions=["instrument", "day_obs"],
         storageClass="DataFrame",
-        name="sspVisitInputs"
+        name="visit_summary_dayobs_14"
     )
     sspTrackletSources = connectionTypes.Output(
         doc="sources that got included in tracklets",
-        dimensions=["instrument"],
-        storageClass="DataFrame",
-        name="sspTrackletSources"
+        dimensions=["instrument", "day_obs"],
+        storageClass="ArrowAstropy",
+        name="ssp_tracklet_sources"
     )
     sspTracklets = connectionTypes.Output(
         doc="summary data for tracklets",
-        dimensions=["instrument"],
-        storageClass="DataFrame",
-        name="sspTracklets"
+        dimensions=["instrument", "day_obs"],
+        storageClass="ArrowAstropy",
+        name="ssp_tracklets"
     )
     sspTrackletToSource = connectionTypes.Output(
         doc="indices connecting tracklets to sspTrackletSources",
-        dimensions=["instrument"],
-        storageClass="DataFrame",
-        name="sspTrackletToSource"
+        dimensions=["instrument", "day_obs"],
+        storageClass="ArrowAstropy",
+        name="ssp_tracklet_to_source"
+    )
+    sspVisitHeliolincInputs = connectionTypes.Output(
+        doc="visit stats plus observer coordinates formatted for heliolinc",
+        dimensions=["instrument", "day_obs"],
+        storageClass="ArrowAstropy",
+        name="ssp_visit_heliolinc_inputs",
     )
-
-
-def getImageTimeTol():
-    return 2./(24.*3600.)
 
 
 class MakeTrackletsConfig(lsst.pipe.base.PipelineTaskConfig, pipelineConnections=MakeTrackletsConnections):
-    mintrkpts = lsst.pex.config.Field(
+    obscode = Field(
+        dtype=str,
+        default="X05",
+        doc="MPC code of observatory"
+    )
+    mintrkpts = Field(
         dtype=int,
         default=2,
         doc="minimum number of sources to qualify as a tracklet"
     )
-    imagetimetol = lsst.pex.config.Field(
+    imagetimetol = Field(
         dtype=float,
-        default=getImageTimeTol(),
-        doc="Tolerance for matching image time, in days: e.g. 1 second"
+        default=1./(24.*3600.),
+        doc="Tolerance for matching image time, in days (default: 1 second)"
     )
-    maxvel = lsst.pex.config.Field(
+    maxvel = Field(
         dtype=float,
         default=1.5,
         doc="Default max angular velocity in deg/day."
     )
-    minvel = lsst.pex.config.Field(
+    minvel = Field(
         dtype=float,
         default=0,
         doc="Min angular velocity in deg/day"
     )
-    exptime = lsst.pex.config.Field(
+    exptime = Field(
         dtype=float,
         default=30,
-        doc="FIXME WITH GOOD DOCUMENTATION. Exposure time"
+        doc="Default exposure time (overriden by sspVisitInputs)"
```

Comment:
It's worth adding that the unit is seconds. Maybe: `doc="Default exposure time in seconds (overriden by sspVisitInputs)"`